weft/handlers.go

```diff
@@ -308,11 +323,14 @@ func setBestPracticeHeaders(w http.ResponseWriter, r *http.Request, customCsp ma
 				s = v1
 			}
 		}
-
 		csp.WriteString(k)
 		csp.WriteString(" ")
-		csp.WriteString(s)
-		csp.WriteString("; ")
+		if k == "script-src" && nonce != "" { //add nonce to CSP
+			csp.WriteString(fmt.Sprintf("'nonce-%s' 'strict-dynamic';", nonce))
```

Comment:
Just checking if you need a space after the semicolon like on line 332.

Reply:
no really necessary, but it leaves a space between different `src` items (last one is redundant)